### ISSUE TITLE
How change the beckend and the label

### ISSUE THREAD

**Issue by vitorbds** [keras-team/keras-io#1109]

**Comment by vitorbds** [comment]
@ksalama  @srihari-humbarwadi Is not possible use pre-trained model like imaginet ? The input shape should be None None,3 ?

**Comment by dhantule** [comment]
Hi @vitorbds, Thanks for reporting this. 

Could you please specify the details of this `cls_model/model.h5` that you're trying to load, also some standalone code to reproduce the above error ? Thank you!
